.github/workflows/build.yml

```
@@ -88,3 +88,30 @@
+    needs:
+      [lint,  build, test, test-storybook]
+    if: github.event_name != 'pull_request' && contains('
+      refs/heads/production
+      refs/heads/alpha
+      refs/heads/beta
+      refs/heads/rc
+      ', github.ref)
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Use Node.js ${{ matrix.node-version }}
```

actionlint

property "node-version" is not defined in object type {}

(expression)